src/blog/dark-mode-with-style-dictionary/index.mdx

```diff
@@ -0,0 +1,1352 @@
+<link rel="stylesheet" href="/dark.css" media="(prefers-color-scheme: dark)" />
+```
+
+You can use the media query to show different images:
+
+```html
+<picture>
+  <!-- Use this image if the user's OS setting is light or unset -->
+  <source srcset="photo-light.png" media="(prefers-color-scheme: light) or (prefers-color-scheme: no-preference)">
+  <!-- Use this image if the user's OS setting is dark -->
+  <source srcset="photo-dark.png" media="(prefers-color-scheme: dark)">
+  <img src="photo-light.png" />
+</picture>
+```
+
+You can use the media query in Javascript too:
+
+```javascript
+if (window.matchMedia && window.matchMedia(`(prefers-color-scheme: dark)`).matches) {
+  // dark mode
+} else {
+  // light mode
+}
+
+// You can add an event listener to do something when the media query changes
+// because a user changed their preference
+var darkQuery = window.matchMedia('(prefers-color-scheme: dark)');
+darkQuery.addListener(function(e) {
+  var newTheme = e.matches ? 'dark' : 'light';
+  // do something here
+});
+```
+
+If you wanted, you could write completely separate sets of styles for light and dark mode for each component. But that sounds a bit tedious and error prone. Instead, people tend to use [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) (sometimes called CSS variables) in addition to the `prefers-color-scheme` media query:
+
+```css
+:root {
+  --color-background-primary: #fff;
+  --color-font-primary: #000;
+}
+
+.card {
+  background-color: var(--color-background-primary);
+  color: var(--color-font-primary);
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --color-background-primary: #000;
+    --color-font-primary: #fff;
+  }
+}
+```
+
+Using CSS custom properties and having a layer of semantic tokens allows you to not have to re-style every component for light and dark mode. The above example is very simple, but can be expanded to a fit the needs of a full design system. Also if you are using inline SVG (as opposed to using SVG in an `<img>` tag), your SVG can be styled with CSS and can use CSS custom properties as well! One more quick tip before we move on: you can use the `currentColor` property in SVG as well to have the graphic inherit its color from its parent. This is useful if you have an icon inside a link or button for example. 
+
+```html
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">
+  <path d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
+    fill="var(--color-font-primary)"/>
+  <path d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
+    fill="currentColor"/>
+</svg>
+```
+
+[See this MDN article for more information about the prefers-color-scheme media query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)
+
+This Style Dictionary demo will create CSS files with CSS custom properties and media queries as well as light and dark SVG graphics. 
```

Comment:
and media queries as well as light and dark SVG graphics. > and media queries, including light and dark SVG graphics.